library/src/morpheus-master/sass/modules/tool/authz/_authz.scss

```diff
@@ -1,19 +1,24 @@
-.checkGrid th:hover,.checkGrid td h4:hover {
-  color: #369 !important;
+.checkGrid th:hover,
```

Comment:
Here's how it looks afterwards. TAs don't work propertly because I haven't pulled in #8733 into my local yet.

![SAK-44348-permissions-after](https://user-images.githubusercontent.com/3632822/97338832-36c4fc00-1858-11eb-9b65-f9f3293a70d5.gif)